index.js

```diff
@@ -28,6 +30,9 @@ module.exports = {
     });
 
     let apolloClient = webpackDependency('apollo-client', {
+      plugins: [
+        new InlineEnvironmentVariables({ NODE_ENV: EmberApp.env() }, { warnings: false })
```

Comment:
The `warnings: false` is necessary here because the plugin warns about inlining an undefined environment variable even if you're passing in an explicit value for it 😒